coral-trino/src/test/java/com/linkedin/coral/trino/rel2trino/TestUtils.java

```diff
@@ -378,6 +378,42 @@ public static void initializeTablesAndViews(HiveConf conf) throws HiveException,
 
     run(driver, "CREATE TABLE test.table_with_binary_column (b binary)");
 
+    run(driver,
```

Comment:
Note to self: add a test where a `select *` is made between two tables:
- table one containing a char, and an int column 
- table two containing a varchar and an int column


`select * from table1 union all select * from table2` 

Not sure whether in this case the cast for the char column should probably kick and rewrite the `select *` to selecting all the columns by name from table1

Reply:
Already handled via the test query:

```
CREATE VIEW IF NOT EXISTS test.fuzzy_union_view_single_branch_evolved AS
SELECT * from test.tableB union all SELECT * from test.tableC
```